spec/mongoid/association/depending_spec.rb

```diff
@@ -849,8 +849,11 @@ class DoubleAssocThree < DoubleAssoc; end
 
       it 'adds an error to the parent object' do
         expect(person.delete).to be(false)
-        expect(person.errors[:restrictable_posts].first).to be(
-          Mongoid::Association::Depending::RESTRICT_ERROR_MSG)
+
+        key_message = "#{Mongoid::Errors::MongoidError::BASE_KEY}.restrict_with_error_dependent_destroy"
+        expect(person.errors[:base].first).to eq(
+          ::I18n.translate(key_message, association: :restrictable_posts)
```

Comment:
This assertion should use the actual expanded string, so that it is clear what the message produced looks like. Right now one has to run the code to determine what the message is.